src/util.ts

```diff
@@ -421,3 +421,56 @@ interface IUploadResult {
     mxcUrl: string;
     size: number;
 }
+
+// Type type
+type Type = Function;  // tslint:disable-line ban-types
+
+/**
+ * Returns true if `obj` is subtype of at least one of the given types.
+ */
+export function instanceofsome(obj: object, types: Type[]): boolean {
+    return types.some((type) => obj instanceof type);
+}
+
+/**
+ * Append the old error message to the new one and keep its stack trace.
+ * Example:
+ *     throw wrap(e, HighLevelError, "This error is more specific");
+ */
+export function wrap<T extends Error>(
+    oldError: Type,
+    newErrorType: new (...args: any[]) => T,  // tslint:disable-line no-any
+    ...args: any[]  // tslint:disable-line no-any trailing-comma
+): T {
+    const newError = new newErrorType(...args);
+    let appendMsg;
+    if (oldError instanceof Error) {
+        appendMsg = oldError.message;
+        newError.stack = oldError.stack;
+    } else {
+        appendMsg = oldError.toString();
+    }
+    newError.message += ":\n" + appendMsg;
+    return newError;
+}
+
+/**
+ * Returns the guild and channel of the given remote room extracted from its ID.
+ * @param remoteRoom The room from which to get the guild and channel.
+ * @returns (guild, channel)-tuple.
+ */
+export function guildAndChannelOf(remoteRoom: RemoteRoom): [string, string] {
```

Comment:
I can only see this being used once, and tbh it's too small in my opinion to make a function in it's own right.

Reply:
Made it a function with module scope.